cli/cmd/sync2.go

```diff
@@ -44,50 +44,50 @@ func syncConnectionV2(ctx context.Context, cqDir string, sourceClient *clients.S
 			word = "table"
 		}
 		if noMigrate {
-			fmt.Printf("Source %s will sync %d %s.\n", sourceSpec.Name, tableCount, word)
+			fmt.Printf("Source %s will sync %d %s.\n", sourceSpec.VersionString(), tableCount, word)
 		} else {
-			fmt.Printf("Source %s will migrate and sync %d %s.\n", sourceSpec.Name, tableCount, word)
+			fmt.Printf("Source %s will migrate and sync %d %s.\n", sourceSpec.VersionString(), tableCount, word)
 		}
 	}
 
 	if !noMigrate {
-		fmt.Println("Starting migration for:", sourceSpec.Name, "->", sourceSpec.Destinations)
-		log.Info().Str("source", sourceSpec.Name).Strs("destinations", sourceSpec.Destinations).Msg("Start migration")
+		fmt.Printf("Starting migration for:%s->%s\n", sourceSpec.VersionString(), destinationStrings)
```

Comment:
`Printf` is less error prone as our linter errors if you use `%s` and don't pass a string